core/src/main/scalajvm/sttp/client3/SttpClient.scala

```diff
@@ -0,0 +1,39 @@
+package sttp.client3
+
+/** A simple, synchronous http client. Usage example:
+  *
+  * {{{
+  * import sttp.client3.{SttpClient, UriContext, basicRequest}
+  *
+  * val client = SttpClient()
+  * val request = basicRequest.get(uri"https://httpbin.org/get")
+  * val response = client.send(request)
+  * println(response.body)
+  * }}}
+  *
+  * Wraps a [[SttpBackend]], which can be substituted or modified using [[wrapBackend]], adding e.g. logging.
+  *
+  * Creating a client allocates resources, hence when no longer needed, the client should be closed using [[close]].
+  */
+case class SttpClient(backend: SttpBackend[Identity, Any]) {
```

Comment:
I have some second thoughts about the name of this client. I wonder if it could be better to somehow have the `Http` word in it.  The role expressiveness could be higher that way - The ideal situation is when somebody entirely unfamiliar with the library knows what a snippet does. `SttpClient` could indicate that it is a client for `STTP` protocol. Looking at our other options: `SttpHttpClient` is a bit hard to read and understand. `HttpClient` is taken by Java and could be misleading, but for example in os-lib there are types like File and it does not pose many problems. I am not entirely sure about it though, it's a tough call.

Reply:
Yeah naming is hard ;) There's also `SimpleHttpClient` which would indicate that the more "advanced" use cases require a different mechanism

Reply:
I agree, I think that we can go with the `SimpleHttpClient` or maybe even `BasicHttpClient`.